examples/albef/configs/vqa.yaml

```diff
@@ -0,0 +1,71 @@
+hidden_size: &hidden_size 768
```

Comment:
Yeah the `_target_` approach is specific to Hydra. It could be a nice extension but for now I personally would not consider it mandatory. I think `&` is just used to mark an alias that can be reused later (i.e. if we want to use the same value for multiple configurables to constrain their values to match)